docs/content/querying/lookups.md

```diff
@@ -296,3 +296,157 @@ To test this setup, you can send key/value pairs to a kafka stream via the follo
 ```
 
 Renames can then be published as `OLD_VAL->NEW_VAL` followed by newline (enter or return)
+
+Dynamic configuration
+---------------------
+
+The following documents the behavior of the cluster-wide config which is accessible through the coordinator.
+The configuration is propogated through the concept of "tier" of servers.
+A "tier" is defined as a group of services which should receive a set of lookups.
+For example, you might have all historicals be part of `__default`, and Peons be part of individual tiers for the datasources they are tasked with.
+
+These configs are accessed using JSON through the following URI template
+
+```
+http://<COORDINATOR_IP>:<PORT>/druid/coordinator/v1/lookups/{tier}/{id}
+```
+
+All URIs below are assumed to have `http://<COORDINATOR_IP>:<PORT>` prepended.
+
+If you have NEVER configured lookups before, you MUST post an empty json object `{}` to `/druid/coordinator/v1/lookups` to initialize the configuration.
```

Comment:
My concern is that if the connection to the DB is temporarily down when a node starts, it is possible that the local node has no knowledge of configuration. If we take an "update" object, and allow it to override a non-existent config, it is not clear to me that the old config would be preserved whenever the DB comes back up.

Having this is kind of a "safety check" to make sure that you really are disregarding whatever may have been in the config previously.

Per cluster it should only ever have to be done ONCE, and after that any occurrence of an empty config in the coordinator is an error.